infra/registry/providers.tf

```diff
@@ -1,8 +1,38 @@
 terraform {
   required_providers {
     openstack = {
-      source = "terraform-provider-openstack/openstack"
+      source  = "terraform-provider-openstack/openstack"
       version = "1.52.1"
     }
   }
-}
\ No newline at end of file
+}
+
+variable "instance_count" {
+  type    = number
+  default = 2
+}
+
+variable "image" {
+  type = string
+  default = "Ubuntu 22.04.3 LTS"
+}
+
+variable "flavor" {
+  type = string
+  default = "m2s.small"
+}
+
+variable "network" {
+  type = string
+  default = "network_1"
+}
+
+variable "subnet" {
+  type = string
+  default = "subnet_1"
+}
+
+variable "router" {
+  type = string
+  default = "router_1"
+}
```

Comment:
what are these variables for? set network, subnet, router and other variables to the same name (`registry` in this case)
Like this: https://github.com/mekstack/mekstack/blob/009d31f75bde17f6dde9a0c3771dad6944e5e30b/infra/sneedaas/module_config.tf#L10

Add a variable for the keypair as well